shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -1322,4 +1322,12 @@ object GlutenConfig {
         "if files are mutable, i.e. file content may change while file path stays the same.")
       .booleanConf
       .createWithDefault(false)
+
+  val CACHE_WHOLE_STAGE_TRANSFORMER_CONTEXT =
+    buildConf("spark.gluten.sql.cacheWholeStageTransformerContext")
+      .internal()
+      .doc("When true, `WholeStageTransformer` will cache the `WholeStageTransformerContext` " +
+        "when executing")
```

Comment:
I'm not clear on this config, maybe need to add some extra words here?
like caching `WholeStageTransformerContext` for xxxx

Reply:
add some words to describle the use case.